selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/NavigateToFileTest.java

```diff
@@ -162,6 +179,32 @@ public void checkNavigateFileFunctionWithJustCreatedFiles() throws Exception {
     editor.closeFileByNameWithSaving(FILE_CREATED_FROM_CONSOLE);
   }
 
+  @Test
+  public void checkNavigateToFileFunctionWithFilesFromHiddenFolders() {
+    projectExplorer.waitProjectExplorer();
+    projectExplorer.waitItem(PROJECT_NAME);
+    projectExplorer.selectItem(PROJECT_NAME);
+    menu.runCommand(
+        TestMenuCommandsConstants.Git.GIT, TestMenuCommandsConstants.Git.INITIALIZE_REPOSITORY);
+    askDialog.acceptDialogWithText(
+        "Do you want to initialize the local repository " + PROJECT_NAME + "?");
+    loader.waitOnClosed();
+    git.waitGitStatusBarWithMess(TestGitConstants.GIT_INITIALIZED_SUCCESS);
+
+    // check that HEAD file from .git folder is not appear in list of found files
+    menu.runCommand(
+        TestMenuCommandsConstants.Assistant.ASSISTANT,
+        TestMenuCommandsConstants.Assistant.NAVIGATE_TO_FILE);
+    navigateToFile.waitFormToOpen();
+    loader.waitOnClosed();
```

Comment:
Waiting of closing of loader looks useless here after the NavigateToFile form had opened, doesn't it?